### DISCUSSION TITLE
Regarding the error when starting the nuxt project.

### DISCUSSION THREAD

**Discussion by toriumi72** [nuxt/nuxt#19924] [Questions]
I followed the instructions in the documentation and when I run "npm i" I get the error "ERROR defineNuxtConfig is not defined" and can't get it to work. i think it is a problem with the nuxt.config.ts file, but I don't know how to solve it. It may be a very simple and comical problem, but please help me if you can. Also, I am getting npm warn.

**Comment by toriumi72** [comment]
I figured out it was because the .nuxt folder didn't exist.

**Reply by toriumi72** [reply]
This is not relevant as it is configured at build time. It just doesn't exist because it hasn't been built. I am not sure how to resolve this.

**Comment by MekarStudio** [comment]
I had this exact same issue few weeks ago. Still not sure what's the reason behind this.

My workaround is downgrading `nuxt` dependency to `3.1.2`  and it should work normally.

 `package.json`
```
  "devDependencies": {
    "nuxt": "3.1.2"
  }
```

and then delete your `node_modules` & `package.lock` files

finally run `npm install`

**Reply by toriumi72** [reply]
It worked fine! But why am I getting an error?

**Comment by toriumi72** [comment]
Thank you very much. You are God to me. I will try it right after this.
ありがとう！！
